node_modules/oae-core/filepreview/css/filepreview.css

```diff
@@ -83,23 +83,25 @@
     }
 }
 
+/* Video.js overrides. These are required to align the video
+ element with the comment widget and to get it to resize correctly */
+
 .video-js {
     position: unset;
     background-color: #fff;
 }
 
-.vjs-tech {
-    background-color: #000;
-    width: 98% !important;
-    left: 20px !important;
+#filepreview-video-container .video-js .vjs-control-bar {
+    width: 98%;
```

Comment:
I'm nervous about the 98% width as it feels like it might not always align well with the comments widget. My preference would be to find an approach where we just add the appropriate margin to the video player container and then let it take up the space it has available.